multisense_ros/src/camera.cpp

```diff
@@ -328,6 +340,15 @@ Camera::Camera(Channel* driver, const std::string& tf_prefix) :
     raw_cam_config_pub_ = calibration_nh_.advertise<multisense_ros::RawCamConfig>(RAW_CAM_CONFIG_TOPIC, 1, true);
     histogram_pub_      = device_nh_.advertise<multisense_ros::Histogram>(HISTOGRAM_TOPIC, 5);
 
+    // TODO(drobinson): guard this inside appropriate conditional
```

Comment:
I'm not sure whether I should advertise these streams by default.

Reply:
I would check the device info and at least make sure the camera is a S27/S30